explore/src/main/scala/explore/targeteditor/AladinContainer.scala

```diff
@@ -114,8 +113,8 @@ object AladinContainer extends AladinCommon {
       // View coordinates base coordinates with pm correction + user panning
       .useStateBy: (p, baseCoordinates) =>
         baseCoordinates.value._1.value.offsetBy(Angle.Angle0, p.options.viewOffset)
-      // Update coordinates if asterism or vizTime changes
-      .useEffectWithDepsBy((p, _, _) => (p.asterism, p.vizTime)):
+      // Update coordinates if asterism or obsTime changes
+      .useEffectWithDepsBy((props, _, _) => (props.asterism, props.obsTime)):
```

Comment:
So this allows a re render anytime obsTime changes

Reply:
Yes, I wasn't sure whether to use the discretized time here. But it's a cheap computation anyway.

Reply:
yes, cheaper than ags